pkg/controller/v1beta1/inferenceservice/components/predictor.go

```diff
@@ -223,6 +265,19 @@ func (p *Predictor) Reconcile(isvc *v1beta1.InferenceService) error {
 		}
 		isvc.Status.PropagateStatus(v1beta1.PredictorComponent, status)
 	}
+	statusSpec, _ := isvc.Status.Components[v1beta1.PredictorComponent]
+	if rawDeployment {
+		podLabelValue = constants.GetRawServiceLabel(constants.DefaultPredictorServiceName(isvc.ObjectMeta.Name))
+	} else {
+		podLabelValue = statusSpec.LatestCreatedRevision
+	}
+	podList, err := isvcutils.ListPodsByLabel(p.client, isvc.ObjectMeta.Namespace, podLabelKey, podLabelValue)
+	if err != nil {
+		return errors.Wrapf(err, "fails to list inferenceservice pods by label")
+	}
+	if len(podList.Items) > 0 {
```

Comment:
nit: move the > 0 check into `PropagateModelStatus`

Reply:
@yuzisun updated, Thanks!